config/maintainers.json

```diff
@@ -22,10 +22,10 @@
     },
     {
       "github_username": "NobbZ",
-      "show_on_website": false,
+      "show_on_website": true,
       "alumnus": false,
-      "name": null,
-      "bio": null,
+      "name": "Norbert Melzer",
+      "bio": "Found out about erlang via exercism, got hooked up and after fixing some trivial bugs, I have been added to the team,",
```

Comment:
Do you mean _hooked_ rather than _hooked up_ here? They're both idiomatic phrases, the first means something like "I liked it so much I couldn't stop" and the second means "I got connected with [someone or something]".

I'd be curious to hear a little bit about what you feel like your day-to-day or overall contributions to the Erlang track have been, and maybe what you like about the language.